lib/themes/dosomething/paraneue_dosomething/templates/system/partials/navigation.tpl.php

```diff
@@ -35,11 +35,11 @@
           <a href="#" class="navigation__dropdown-toggle"><?php print $user_identifier ?></a>
           <ul>
             <li><?php print l("My Account", 'user/'. $user->uid); ?></li>
-            <li><a id="link--logout" href="<?php print $front_page; ?>user/logout"><?php print t('Log Out'); ?></a></li>
+            <li><?php print l(t('Log Out'), 'user/logout', array('attributes' => array('class' => array('secondary-nav-item')))) ?></li>
```

Comment:
I can see it here: https://github.com/DoSomething/phoenix/blob/dev/tests/integration/user/login.js#L28